tensorboard/webapp/widgets/linked_time_fob/linked_time_fob_component.ng.html

```diff
@@ -0,0 +1,29 @@
+<!--
+@license
+Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<div (dblclick)="typeStepRequested()">
```

Comment:
However you are tracking tasks, consider adding this as something to examine (if you haven't already):

I'm a little concerned about how these fobs are interactable but do not have any markup describing that (especially for accessibility reasons). There is nothing about the markup that describes them as being click and draggable, for example, nor about being double clickable.

The use of \<input\> is good but that only happens AFTER they've been double clicked.

Otherwise, are they buttons? https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role.

Can we somehow label the click-and-drag functionality? Or do we ignore click-and-drag from accessibility stand point?

How do other products with similar functionality deal with this?